internal/adapter/lsp/document.go

```diff
@@ -194,11 +197,25 @@ func (d *document) DocumentLinkAt(pos protocol.Position) (*documentLink, error)
 	return nil, nil
 }
 
+// Recursive function to check whether a link is within inline code block.
+func linkWithinInlineCode(strBuffer string, linkStart, linkEnd int, insideInline bool) bool {
+	if backtickId := strings.Index(strBuffer, "`"); backtickId > 0 && backtickId < linkEnd {
```

Comment:
`backtickId > 0` needs to be `backtickId >= 0` to account for links within backticks being the first instance of a string on the line.

Reply:
Huh, don't know what I was doing there. I think the equal sign got dropped accidentally when changing this function for the thousandth time :laughing: